packages/nextjs-kit/src/__tests__/snapshotTests/contentWithImage.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render } from '@testing-library/react';
+import ContentWithImage from '../../components/contentWithImage';
+import examplePostData from '../data/examplePostData.json';
+import { vi } from 'vitest';
+
+vi.mock('next/image', () => ({
+	__esModule: true,
+	default: (props: any) => {
+		return <img {...props} />;
+	},
+}));
```

Comment:
It is likely that we will need to mock `next/image` anytime we use `next/image`, can we move this out to `__mocks__` like we have in the `next-drupal-starter`?